<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dividendos</class>
 <widget class="QDialog" name="Dividendos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dividendos</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>671</width>
     <height>421</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SÃ­mbolo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome da Epresa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data Ex</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor do Dividendo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data de Pagamento</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Ex</string>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>tableWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
